PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -548,12 +548,11 @@
 		rm "DietPi-$GITBRANCH/dietpi/pre-patch_file"
 		rm "DietPi-$GITBRANCH/dietpi/patch_file"
 
+		l_message='Set execute permissions for DietPi scripts' G_RUN_CMD chmod +x -R "DietPi-$GITBRANCH"
```

Comment:
@Fourdee 
Little complain: We will set execute permissions to settings files that definitely don't need them. Also the previous solution below was failsafe by (re)setting execute permissions as well to files that might not be included in the update.

/etc/profile.d/dietpi-*.sh /etc/bashrc.d/dietpi-*.sh actually not require execute permissions, since they are sourced.